lib/ui/painting.dart

```diff
@@ -5282,6 +5282,10 @@ class Canvas extends NativeFieldWrapperClass1 {
     _drawRRect(rrect._getValue32(), paint._objects, paint._data);
   }
 
+  // RadiusX and radiusY values, on every corner, must be finite.
+  // NaN values and infinities are not valid.
+  // Negative radii are interpreted as 0.
+  // Positive radii are interpreted as 0 < radii < float32.
```

Comment:
I think you can just say `radii are clamped between 0..FLT_MAX` or something like that.

The problem ends up happening upsream of this method if someone provides a value > FLT_MAX.